File: JavaScript/1074.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var teste = parseInt(lines.shift()); 

for (var i = 0; i < teste ; i++) {
  var num = parseInt(lines.shift());
    
  if(num<0){
    if(num%2==0){
      console.log("EVEN NEGATIVE")
    }
    else{
      console.log("ODD NEGATIVE")  
    }    
  }
  else if(num>0){
    if(num % 2 == 0){
      console.log("EVEN POSITIVE")
    }
    else{
      console.log("ODD POSITIVE")
    }
  }
  else if(num==0){
    console.log("NULL")
  }
}   

File: Ruby/1074.rb
teste = gets.chomp.to_i

i = 0

while i < teste
    num = gets.chomp.to_i

    if num < 0
        if num % 2 == 0
            puts "EVEN NEGATIVE"
        else
            puts "ODD NEGATIVE"
        end

    elsif num > 0
        if num % 2 == 0
            puts "EVEN POSITIVE"
        else
            puts "ODD POSITIVE"
        end

    elsif num == 0
        puts "NULL"
    end
    
    i = i + 1
end